docs/06-concepts/18-testing/03-advanced-examples.md

```diff
@@ -0,0 +1,98 @@
+# Advanced examples
+
+## Test business logic that depends on `Session` {#business-logic-depends-on-session}
+
+It is common to break out business logic into modules and keep it separate from the endpoints. If such a module depends on a `Session` object (e.g to interact with the database), then the `withServerpod` helper can still be used and the second `endpoint` argument can simply be ignored:
+
+```dart
+withServerpod('Given decreasing product quantity when quantity is zero', (
+  sessionBuilder,
+  _ /* Ignore */,
```

Comment:
I think the that `_` is generally known to be ignored (I'd propose we skip the comment here).

Reply:
Good point!